### ISSUE TITLE
Training Detail

### ISSUE THREAD

**Issue by neverneverendup** [allenai/scifact#3]
Dear author,
Thanks very much for your excellent work. I want to train a model according to the default parameters setting in 'rationale_selection_scifact_train.py', but what I meet OOM. I used a single GTX 1080ti for training, and the batch is set to 8. Could you plz tell me your hardware settings and the time cost to train a single model? 
BTW,when testing the pre-trained model according to the full_example script, the results I obtained  is slightly different with the result in table1. Is it normal?
![image](https://user-images.githubusercontent.com/41896224/82558897-43567180-9ba1-11ea-9e21-5f8ea729632b.png)

Thanks~

**Comment by PeterL1n** [comment]
1. We used 1 P100 GPU with 16G of VRAM. The training is quite fast on the SciFACT dataset. It can finish all 20 epochs in about an hour.

2. The paper table 1 reports metrics on the test set. We currently don't publish the test set labels so I am guessing that your metrics are performed on the dev set?

**Comment by neverneverendup** [comment]
Thanks😊! I got it.
